podman/api_connection.py

```diff
@@ -81,6 +81,16 @@ def request(self,
             )
         return response
 
+    def make_call(self, api, endpoint, method='GET', body=None, **kwargs):
```

Comment:
Check the current state of the apiconnection, we have a few helper functions specifically for post/get/delete

Reply:
Fair enough.  A little frustrating that this sat for 5+ months, but now I have to rebase...


I'll close this PR and resubmit.